x-pack/plugins/security_solution/common/endpoint/types.ts

```diff
@@ -41,14 +41,30 @@ type ImmutableMap<K, V> = ReadonlyMap<Immutable<K>, Immutable<V>>;
 type ImmutableSet<T> = ReadonlySet<Immutable<T>>;
 type ImmutableObject<T> = { readonly [K in keyof T]: Immutable<T[K]> };
 
+export interface EventStats {
+  /**
+   * The total number of related events (all events except process and alerts) that exist for a node.
+   */
+  total: number;
```

Comment:
Is this going to be used by the UI?

Reply:
It will, in the mocks we have total counts on the host buttons